chronix-server-test-integration/src/main/java/de/qaware/chronix/solr/CSVImporter.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (C) 2016 QAware GmbH
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package de.qaware.chronix.solr;
+
+import com.google.common.collect.Lists;
+import de.qaware.chronix.ChronixClient;
+import de.qaware.chronix.timeseries.MetricTimeSeries;
+import org.apache.solr.client.solrj.SolrClient;
+import org.apache.solr.client.solrj.SolrQuery;
+import org.apache.solr.client.solrj.impl.HttpSolrClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.UnsupportedEncodingException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.text.DecimalFormat;
+import java.text.NumberFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * Simple csv reader for test purposes
+ *
+ * @author f.lautenschlager
+ */
+public class CSVImporter {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CSVImporter.class);
+
+    public static final List<String> LIST_STRING_FIELD = Lists.newArrayList("List first part", "List second part");
+    public static final List<Integer> LIST_INT_FIELD = Lists.newArrayList(1, 2);
+    public static final List<Long> LIST_LONG_FIELD = Lists.newArrayList(11L, 25L);
+    public static final List<Double> LIST_DOUBLE_FIELD = Lists.newArrayList(1.5D, 2.6D);
+
+    public static byte[] bytes;
```

Comment:
\[Checkstyle\] ERROR: Variable 'bytes' must be private and have accessor methods\.